packages/datadog-instrumentations/src/kafkajs.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const producerStartCh = channel('apm:kafkajs:produce:start')
+const producerMessageCh = channel(`apm:kafkajs:produce:message`)
```

Comment:
It looks like `start` already gets all messages. Is this needed?

Reply:
I'd probably also reorganize these to group them by `produce` and `consume` for readability.